backstop/tests/table.html

```diff
@@ -273,15 +273,66 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-slot">
               <button
-                class="iui-button iui-borderless"
+                class="iui-button iui-borderless iui-dropdown"
```

Comment:
Instead, you can add custom class and in this html set `position: relative` (custom class will help you to select those elements too).. This should help you using `position: absolute` put the menu in the correct place.